src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/StepDetailsBlock/StepDetailsBlock.tsx

```diff
@@ -21,10 +21,19 @@ const StepDetailsBlock: FC<StepDetailsBlockProps> = ({ text, buttonProps }) => (
       {
         key: 'fund',
         content: (
-          <Button
-            {...buttonProps}
-            className={clsx(buttonProps.className, 'w-full')}
-          />
+          <div className="flex w-full justify-center">
+            {'actionType' in buttonProps ? (
```

Comment:
could we please make this composable instead of checking props. All we're doing is rendering a button with the provided props, a perfect use case to just pass it already rendered to this component